src/codec/RowWriterV2.cpp

```diff
@@ -916,7 +916,10 @@ std::string RowWriterV2::processOutOfSpace() noexcept {
 }
 
 
-WriteResult RowWriterV2::finish() noexcept {
+WriteResult RowWriterV2::finish(int64_t ts) noexcept {
```

Comment:
Why not use the actual timestamp? We don't need to modify it in later version.

Reply:
Good idea.